resources/run_adf.sh

```diff
@@ -6,13 +6,13 @@
 # This script will start a local mongod and Atlas Data Federation instance, used for integration testing.
```

Comment:
Most of these changes are from PR 12 to change how `go` is called.  I added a few similar changes for the `jq` command as the evergreen host has trouble finding it in the path.